File: ytrue-bean/src/main/java/com/ytrue/bean/resp/system/SysJobListResp.java
package com.ytrue.bean.resp.system;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;

import java.io.Serial;
import java.io.Serializable;

@Data
public class SysJobListResp implements Serializable {

    @Serial
    private static final long serialVersionUID = 1L;


    @Schema(description = "唯一标识")
    private Long id;

    @Schema(description = "岗位名称")
    private String jobName;

    @Schema(description = "岗位排序")
    private Long jobSort;


    @Schema(description = "是否启用")
    private Boolean status;
}

File: ytrue-admin-server-csharp/Ytrue.Models/Resps/System/SysJobListResp.cs
namespace Ytrue.Models.Resps.System;

public class SysJobListResp
{
    /// <summary>
    /// ID
    /// </summary>
    public long Id { get; set; }

    /// <summary>
    /// 岗位名称
    /// </summary>
    public string JobName { get; set; }

    /// <summary>
    /// 岗位状态:0=禁用,1=启用
    /// </summary>
    public int Status { get; set; }

    /// <summary>
    /// 排序
    /// </summary>
    public int? JobSort { get; set; }
}